README.md

```diff
@@ -1,12 +1,7 @@
 oc-template-handlebars
 ======================
 
-[![Greenkeeper badge](https://badges.greenkeeper.io/opencomponents/oc-template-handlebars.svg)](https://greenkeeper.io/)
-
-[![Build Status](https://travis-ci.org/opencomponents/oc-template-handlebars.svg?branch=master)](https://travis-ci.org/opencomponents/oc-template-handlebars)
-
-This is a module for handling [Handlebars](https://github.com/wycats/handlebars.js) templates in [OpenComponents](https://github.com/opentable/oc).
-It is part of OC frameworks's core.
+Deprecated. Moved to [opencomponents/base-templates](https://github.com/opencomponents/base-templates)
```

Comment:
Perhaps we could point to the specific packages?
- https://github.com/opencomponents/base-templates/tree/master/packages/oc-template-handlebars
- https://github.com/opencomponents/base-templates/tree/master/packages/oc-template-handlebars-compilers